.github/workflows/spec.yml

```diff
@@ -0,0 +1,13 @@
+name: Spec
+
+on: push
+
+jobs:
+  spec:
+    name: Run specs
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: oprypin/install-crystal@v1
```

Comment:
No idea who this is but this action seems like the one most commonly used in GH actions: https://github.com/search?l=YAML&q=%22oprypin%2Finstall-crystal%22&type=Code

Reply:
ysbaddaden/